### PR TITLE
gimp: update to 2.10.8

### PR DESCRIPTION
**Pull request by biopsin**

### CODE REVIEW

File: srcpkgs/gimp/template [hunk 1]
```diff
@@ -61,6 +61,6 @@ gimp-python_package() {
 	short_desc+=" - Python2 bindings"
 	pkg_install() {
 		vmove usr/lib/gimp/2.0/python
-		vmove "usr/lib/gimp/2.0/plug-ins/*.py"
+		vmove usr/lib/gimp/2.0/plug-ins
```
Comment (CameronNemo) [inline srcpkgs/gimp/template]: The plugins should stay in the main package!

Comment (biopsin) [reply srcpkgs/gimp/template]: when I build it locally it cound not find any *.py as they are in subfolders

Comment (biopsin) [reply srcpkgs/gimp/template]: looking in the subfolder of plug-ins/pygimp/plug-ins there are some *.py

Comment (biopsin) [reply srcpkgs/gimp/template]: mv: cannot stat '/destdir//gimp-2.10.8/usr/lib/gimp/2.0/plug-ins/*.py': No such file or directory

/plug-ins $ find . -name "*.py"
./py-slice/py-slice.py
./python-console/pyconsole.py
./python-console/python-console.py
./colorxhtml/colorxhtml.py
./file-openraster/file-openraster.py
./python-eval/python-eval.py
./gradients-save-as-css/gradients-save-as-css.py
./foggify/foggify.py
./histogram-export/histogram-export.py
./palette-sort/palette-sort.py
./palette-offset/palette-offset.py
./palette-to-gradient/palette-to-gradient.py

Comment (jnbr) [reply srcpkgs/gimp/template]: You can do something like I did here: https://github.com/jnbr/void-packages/commit/545a737bd7ff2f1110bdd83712fe8e9f5681fcfe or have a look here: https://github.com/void-linux/void-packages/pull/3376

Comment (biopsin) [reply srcpkgs/gimp/template]: there is still issue upgrading from 2.10.4 to this

ERROR: gimp-2.10.8_1: [unpack] failed to extract file `./usr/lib/gimp/2.0/plug-ins/web-browser/web-browser': Not a directory
ERROR: gimp-2.10.8_1: [unpack] failed to extract files: Not a directory
ERROR: gimp-2.10.8_1: [unpack] failed to unpack files from archive: Not a directory

had to uninstall old package this fix it

Comment (jnbr) [reply srcpkgs/gimp/template]: This is why 2.10.6 never made it into the repos.

Comment (CameronNemo) [reply srcpkgs/gimp/template]: yes it is currently blocked on void-linux/xbps#14

**Comment by biopsin** [comment]
pkg upgrade stalled, xbps-install issue https://github.com/void-linux/xbps/issues/14

**Pull request closed without merging**